Leagues_Data_and_Adaptations.py

```diff
@@ -66,6 +22,7 @@ def df_creator(path, file):
               'season-1819_csv.csv']
 laLiga0919Concatenated = pd.concat([df_creator(la_liga_path, file) for file in files_list])

```

Comment:
good implementation. 
variable name is super long, let's try and shorten it.
BTW, I didn't see you import pandas (only to_numeric)